en/telemetry/telemetry_wifi.md

```diff
@@ -0,0 +1,20 @@
+# WiFi Telemetry Radio
+
+WiFi telemetry enables MAVLink communication between a WiFi radio on a vehicle and a GCS.  WiFi typically offers shorter range than a normal telemetry radio, but supports higher data rates, and makes it easier to support FPV/video feeds. Usually only a single radio unit for the vehicle is needed (assuming the ground station already has WiFi).
+
+PX4 supports telemetry via UDP and Wifi. It broadcasts a heartbeat to port 14550 on 255.255.255.255 until it receives the first heartbeat from a ground control station, at which point it will only send data to this ground control station.
+
+
+## 3DR WiFi Telemetry
+
+The [3DR WiFi Telemetry Radio](https://store.3dr.com/products/wifi-telemetry-radio) is supported by PX4. Simply connect it to the flight controller's `TELEM` port. Once connected to the WiFi, it should automatically connect to *QGroundControl*.
```

Comment:
@bkueng So I just want to confirm the semantics here. You've plugged the wifi into TELEM. I think that what happens is that it creates a wifi hotspot with the hotspot and password below. So on your GCS you just connect to that hotspot. Is that correct? (if so this will need minor update).

Reply:
I think so but I've never used that myself, so I cannot say for sure.